python/docs/source/reference/timestream/aggregation.md

```diff
@@ -15,6 +15,7 @@ Windowed:
 .. autosummary::
    :toctree: ../apidocs/
 
+    Timestream.coalesce
```

Comment:
Coalesce isn't really an aggregation -- it should probably be documented wherever we have `else_` (so `misc.md` for now)?